### PR TITLE
feat(traffic_light_classifier): add build only option for tensort engine

### PR DESCRIPTION
**Pull request by yukke42**
## Description

- the option to build tensorrt engine without inference is required for test on cloud

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by yukkysaito — APPROVED** [review]
LGTM

File: perception/traffic_light_classifier/README.md [hunk 1]
```diff
@@ -54,14 +54,15 @@ These colors and shapes are assigned to the message as follows:
...
-| `input_w`         | str  | the width of an input image          |
+| Name              | Type | Description                                      |
+| ----------------- | ---- | ------------------------------------------------ |
+| `model_file_path` | str  | path to the model file                           |
+| `label_file_path` | str  | path to the label file                           |
+| `precision`       | str  | TensorRT precision, `fp16` or `int8`             |
+| `input_c`         | str  | the channel size of an input image               |
+| `input_h`         | str  | the height of an input image                     |
+| `input_w`         | str  | the width of an input image                      |
+| `build_only`      | bool | shutdown node after TensorRT engin file is built |
```
Comment (wep21) [inline perception/traffic_light_classifier/README.md:65]: @yukke42 Could you fix typo(engin -> engine)?

Comment (yukke42) [reply perception/traffic_light_classifier/README.md:65]: @yukke42 I'll make another PR.

Comment (yukke42) [reply perception/traffic_light_classifier/README.md:65]: <https://github.com/autowarefoundation/autoware.universe/pull/3018>.

**Pull request merged**